spec/controllers/items_controller_spec.rb

```diff
@@ -128,6 +127,46 @@
         end
       end
 
+      context "with an already existing item name" do
+        it "shouldn't create an item with the same name" do
+          post :create, params: default_params.merge(item_params)
+
+          item_params_with_same_name =
+            {
+              item: {
+                name: "Really Good Item",
+                partner_key: create(:base_item).partner_key,
+                value_in_cents: 1001,
+                package_size: 5,
+                distribution_quantity: 30
+              }
+            }
+          post :create, params: default_params.merge(item_params_with_same_name)
+
+          expect(flash[:error]).to eq("An item with that name already exists (could be an inactive item).")
+          expect(response).to render_template(:new)
+        end
+
+        it "shouldn't create an item with the same name, case-insensitively" do
```

Comment:
This is a superset of the previous test - if it fails case-insensitively, it will by definition also fail for case-sensitive.